compiler/ballerina-lang/src/main/java/io/ballerina/projects/PackageDescriptor.java

```diff
@@ -72,9 +72,11 @@ public PackageVersion version() {
         return Optional.ofNullable(repository);
     }
 
-    public boolean isLangLibPackage() {
+    public boolean isBuiltInPackage() {
```

Comment:
This is an API change.  Should we do this now? 

Can we introduce a new method and deprecate the old method later?

Reply:
Decided to add a new `isBuiltInPackage(...)` method and use it only when persisting dependencies to `Dependencies.toml`. This will resolve this.